src/docfx/config/ops/OpsPreProcessor.cs

```diff
@@ -64,7 +64,7 @@ private void LogError(LogItem item)
         {
             if (!string.IsNullOrEmpty(item.Code))
             {
-                _errorLog.Write(new Error(MapLevel(item.MessageSeverity), item.Code, item.Message, new FilePath(item.File), item.Line ?? 0));
+                _errorLog.Write(new Error(MapLevel(item.MessageSeverity), item.Code, item.Message, new FilePath(PathUtility.Normalize(Path.GetRelativePath(_buildOptions.DocsetPath, item.File))), item.Line ?? 0));
```

Comment:
`FilePath` is normalized:
```suggestion
                _errorLog.Write(new Error(MapLevel(item.MessageSeverity), item.Code, item.Message, new FilePath(Path.GetRelativePath(_buildOptions.DocsetPath, item.File)), item.Line ?? 0));
```